arfc-pres.tex

```diff
@@ -26,7 +26,7 @@
 \usepackage{booktabs} % nice rules for tables
 \usepackage{microtype} % if using PDF
 \usepackage{bigints}
-\usepackage{minted}
+%\usepackage{minted}
```

Comment:
why no minted? I fully expect to use code examples in other slides. Will this break your slides?

Reply:
yeah that was to let mine compile

Reply:
ok, well, I'm going to have to add it back .